src/analysis/interpretation/interpret.ts

```diff
@@ -76,6 +78,32 @@ function interpretPayloadChecks(
   };
 }
 
+export function filterDependenciesWarnings(
```

Comment:
I feel like "filter" is too generic, maybe we could name the `filterDependenciesWarnings` function differently so that it reveals the reason behind the filtering like:

- `keepOnlyNonIgnoredDependenciesWarnings`
- `excludeIgnoredDependenciesWarnings`

What's your opinion on that?